app/controllers/project/settings/donations.js

```diff
@@ -76,8 +83,15 @@ export default Controller.extend({
      */
     saveDonationGoal(donationGoal, properties) {
       donationGoal.setProperties(properties);
+      console.log('set');
```

Comment:
When the donation goal `hasDirtyAttributes` it gets sorted to the top of the list for some reason. I tried a number of things to fix this but no dice. In particular, I thought the transform would affect it. This seemed to make no difference.